bench/common/blas_benchmark.hpp

```diff
@@ -372,13 +251,14 @@ std::vector<benchmark_instance<ElemT, ExecutorT, ParamT>*> benchmark_suite();
 template <typename Ex, typename ParamT>
 int main_impl(Range<ParamT>* range_param, const unsigned reps, Ex ex,
               output_type output = output_type::STDOUT) {
+  benchmark<>::output_headers(output);
   auto fbenchmarks = benchmark_suite<float, Ex, ParamT>();
   for (auto b : fbenchmarks) {
     run_benchmark(b, range_param, reps, ex, output);
   }
 
 #ifndef NO_DOUBLE_SUPPORT
```

Comment:
This double negation is a bit confusing. Could we change the macro so is #ifdef  DOUBLE_SUPPORT instead?

Reply:
I think it would be good to create a separate PR for fixing this issue

Reply:
I agree - I'm happy to look at it tomorrow if people want? I think it might be best done after merging the two currently open+active PRs.

Reply:
agreed!